src/Domain/Creators/Cylinder.hpp

```diff
@@ -137,6 +138,14 @@ class Cylinder : public DomainCreator<3> {
         "between LowerBound and UpperBound."};
   };
 
+  struct OuterRadialDistribution {
+    using type = domain::CoordinateMaps::Distribution;
+    static constexpr Options::String help = {
+        "Select the radial distribution of grid points in the outermost "
+        "spherical shell. The 'RadialPartitioning' must have at least one "
+        "entry if you select a distribution other than 'Linear'."};
```

Comment:
This is a cylinder, so it presumably doesn't have spherical shells.  Also in various other places.